test/wdio.conf.ts

```diff
@@ -192,8 +192,7 @@ export const config: Options.Testrunner = {
     mochaOpts: {
         ui: 'bdd',
         timeout: 60000,
-        grep: grep.join('|'),
-        invert: true
```

Comment:
How is this related to the original issue?

Reply:
Not related at all. I did this to run tests in all platforms already set up in Actions. I was going to add a comment to check if there is something wrong about running tests in GitHub actions. Are we fine with it?

Reply:
it is currently causing the test suite to fail because some web tests should not run in CI due to issues with the environment.

Reply:
Reverting.